src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -88,7 +88,7 @@ const DomainList = (props) => {
           testId="images"
           fill={item.karmaScore}
           size={normalize(51)}
-          width={normalize(2.3)}>
+          width={normalize(2)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `width` prop from `normalize(2.3)` to `normalize(2)` might cause visual changes in your application. If this is intended, then it's fine. However, if you want to maintain the previous appearance, you should revert this change.
```diff
- width={normalize(2)}>
+ width={normalize(2.3)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->